Models/ViewModels/RetrievingViewModel.cs

```diff
@@ -1,7 +1,11 @@
-﻿namespace Garage2._0.Models.ViewModels
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Garage2._0.Models.ViewModels
 {
     public class RetrievingViewModel
     {
+        //[RegularExpression(@"^[A-Z]{3}\d{3}$", ErrorMessage = "Please enter a valid registration number with 3 uppercase letters and 3 numbers.")]
```

Comment:
Here is a similar thing, I changed it to ' [RegularExpression("^[A-Z0-9]*$")] ' instead, do we need to enforce it this over here too though?